src/Nethermind/Nethermind.BeaconNode.Test/BeaconNodeWorkerTest.cs

```diff
@@ -138,7 +138,7 @@ public async Task SleepTimeIsCorrectFromStartOfSecond()
             // Assert
             signal.ShouldBeTrue();
             TimeSpan expectedTime = TimeSpan.FromMilliseconds(250);
-            TimeSpan tolerance = TimeSpan.FromMilliseconds(25);
+            TimeSpan tolerance = TimeSpan.FromMilliseconds(40);
```

Comment:
you can use RetryAttribute to avoid problems on the build servers (assuming it is an async / nondeterministic test)

Reply:
I just used MSTest, for simplicity, and I think that is NUnit. Can change over if you want consistency, or TestMethodAttribute is extensible, so I can add a RetryTestMethodAttribute. Otherwise, it is okay for now.